packages/langium/test/workspace/document-builder.test.ts

```diff
@@ -393,7 +397,7 @@ describe('DefaultDocumentBuilder', () => {
             expect(isOperationCancelled(err)).toBe(true);
         }
         expect(document1.state).toBe(DocumentState.IndexedReferences);
-        expect(document2.state).toBe(DocumentState.Linked);
+        expect(document2.state).toBe(DocumentState.IndexedReferences);
```

Comment:
Similar to above: The test was very explicit in what it wanted to achieve; If I abort the build when one element has been reference-indexed, I expect the other one to still be at linked. The document builder should not proceed after `onDocumentPhase` has been called.

Reply:
Already changed it back ⬇️